images/kubekins-e2e/variants.yaml

```diff
@@ -32,7 +32,7 @@ variants:
     BAZEL_VERSION: 3.4.1
     OLD_BAZEL_VERSION: 2.2.0
   '1.27':
-    CONFIG: main
+    CONFIG: '1.27'
```

Comment:
I think main _also_ was added because of pending discussions around master => main

https://github.com/kubernetes/enhancements/pull/3881

Reply:
cc @cpanato

Reply:
oh lol, I see.

Reply:
Thanks for the update!